helmfile.d/charts/gatekeeper/templates/templates/config.yaml

```diff
@@ -9,3 +9,18 @@ spec:
       - group: "networking.k8s.io"
         version: "v1"
         kind: "NetworkPolicy"
+      - group: "apps"
+        version: "v1"
+        kind: "Deployment"
+      - group: "apps"
+        version: "v1"
+        kind: "ReplicaSet"
+      - group: "apps"
+        version: "v1"
+        kind: "StatefulSet"
+      - group: ""
+        version: "v1"
+        kind: "ReplicationController"
+      - group: "policy"
+        version: "v1"
+        kind: "PodDisruptionBudget"
```

Comment:
If you want to keep the kinds mentioned in my previous comment for future work, then please add them here as well :slightly_smiling_face: might save a future headache.

Reply:
I removed the kinds from the constraint.
As extra info: it would actually not be good to add the extra kinds here before they are needed, because that would cause gatekeeper to cache those kinds and increase the resource of the gatekeeper pods.